### PR TITLE
Hw5 may be done

### PR DESCRIPTION
**Pull request by Shlepanets**

### CODE REVIEW

File: hw5.md [hunk 1]
```diff
@@ -1,32 +1,177 @@
...
+    
 1.9 (*) Open webserver webpage in browser of your Host machine of VirtualBox (Windows or Mac or whatever else you use). You may need to setup port forwarding in settings of VirtualBox.
 
+    # On Host machine (Windows)
+    C:\Users\shlep>ssh -L 12347:localhost:12346 mivanov@192.168.56.101
+    mivanov@192.168.56.101's password:
+    Last login: Sun Nov 18 18:24:51 2018 from 192.168.56.1
+    
+    # in Chrome browser, while ssh connection is open
+    http://localhost:12347
```
Comment (alant94) [inline hw5.md]: I think simple `http://192.168.56.101:12346` should work without second ssh tunnel

File: hw5.md [hunk 2]
```diff
@@ -1,32 +1,177 @@
...
+    Nov 18 19:51:22 localhost.localdomain dbus[620]: [system] Successfully activated service 'org.freedesktop.timedate1'
+    Nov 18 19:53:05 localhost.localdomain dbus[620]: [system] Activating via systemd: service name='org.freedesktop.timedate1' unit='dbus-org.freedesktop.timedate1.service'
+    Nov 18 19:53:05 localhost.localdomain dbus[620]: [system] Successfully activated service 'org.freedesktop.timedate1'
+    Nov 18 19:55:14 localhost.localdomain dbus[620]: [system] Activating via systemd: service name='org.freedesktop.timedate1' unit='dbus-org.freedesktop.timedate1.service'
+    Nov 18 19:55:14 localhost.localdomain dbus[620]: [system] Successfully activated service 'org.freedesktop.timedate1'
+
 2.3. Configure rsyslogd by adding a rule to the newly created configuration file /etc/rsyslog.d/auth-errors.conf to log all security and authentication messages with the priority alert and higher to the /var/log/auth-errors file. Test the newly added log directive with the logger command. (multiple commands)
+
+    [root@localhost mivanov]# cat /etc/rsyslog.d/auth-errors.conf
```
[the remark this answers is not in the document]
Comment (Shlepanets) [reply hw5.md]:
```suggestion
    [root@localhost mivanov]# systemctl restart rsyslog
    [root@localhost mivanov]# cat /etc/rsyslog.d/auth-errors.conf
```

[the remark this answers is not in the document]
Comment (Shlepanets) [reply hw5.md]: How about this?

File: hw5.md (hunk 1 above)
Comment (Shlepanets) [reply hw5.md]: I've tried, and it didn't work. I did not find a simple way to setup port forwarding in VirtualBox, so I deside to make another ssh tunnel

**Pull request merged**